lnrpc/rpc.proto

```diff
@@ -2617,6 +2617,12 @@ message PolicyUpdateRequest {
 
     /// If set, the maximum HTLC size in milli-satoshis. If unset, the maximum HTLC will be unchanged.
     uint64 max_htlc_msat = 6 [json_name = "max_htlc_msat"];
+
+    /// The minimum HTLC size in milli-satoshis. Only applied if min_htlc_msat_specified is true.
+    uint64 min_htlc_msat = 7 [json_name = "min_htlc_msat"];
+
+    /// If true, min_htlc_msat is applied.
+    bool min_htlc_msat_specified = 8 [json_name = "set_min_htlc_msat"];
```

Comment:
Would be nice if protobuf supported setting default values...

Reply:
You mean we could have used something like -1 to signal 'not specified'?

Reply:
Why did we never do something like this for the other fields? Using the cli to determine if they were present or not? We could've even added a new message like: 
```proto
message MaybeUint64 {
    uint64 value = 1; 
    bool present = 2; 
}